DQM/EcalMonitorClient/python/MLClient_cfi.py

```diff
@@ -0,0 +1,411 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQM.EcalMonitorTasks.OccupancyTask_cfi import ecalOccupancyTask
+
+#parameters dervied from training
+MLThreshold = 0.1761
+PUcorr_slope = 8625.62354249
+PUcorr_intercept= 354776.97564344
+#Avg Occupancy of 2018 data
+avgOcc = (1.4228867,1.4111477,1.4235557,1.403253 ,1.2926203,1.3110067,1.4317631,
```

Comment:
I am not sure if this is the preferred way to store this large information in CMSSW; I let @qliphy and @perrotta to comment